File: 7 семестр/ПМУ/Water/app/src/main/java/com/example/water/MainActivity.java
package com.example.water;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.opengl.GLSurfaceView;
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        MyRenderer render = new MyRenderer(this);
        super.onCreate(savedInstanceState);
        GLSurfaceView canvas = new GLSurfaceView(this);
        canvas.setEGLContextClientVersion(2);
        canvas.setRenderer(render);
        canvas.setRenderMode(GLSurfaceView.RENDERMODE_CONTINUOUSLY);
        setContentView(canvas);
    }
}

File: 7 семестр/ПМУ/Phong/app/src/main/java/com/example/phong/MainActivity.kt
package com.example.phong

import android.opengl.GLSurfaceView
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val GLView = GLSurfaceView(this)
        GLView.setEGLContextClientVersion(2)
        GLView.setRenderer(MyRenderEngine())
        setContentView(GLView)
    }
}